### PR TITLE
@eessex => [EOY] Analytics + minor mobile stuff

### PR DESCRIPTION
**Pull request by kanaabe**
- Analytics
- Responsive + non-iPad pages should be non-scalable

### CODE REVIEW

File: components/main_layout/templates/head.jade [hunk 1]
```diff
@@ -18,7 +18,7 @@ script( src=asset('/assets/scroll_frame.js') )
 
 //- Set viewport size depending on if the page is responsive or not
 if sd.IS_RESPONSIVE && (userAgent && !userAgent.match('iPad'))
-  meta( name='viewport', content='width=device-width, initial-scale=1' )
+  meta( name='viewport', content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no' )
```
Comment (kanaabe) [inline components/main_layout/templates/head.jade]: i checked out a few pages, but it didn't seem offensive for all `IS_RESPONSIVE` pages to be non-scalable. Lmk if you have other thoughts @craigspaeth 

another option is to set an IS_MOBILE flag using something from [here](https://github.com/artsy/force/blob/master/lib/middleware/redirect_mobile.coffee#L16)

**Comment by eessex** [comment]
Great stuff, thanks for tackling these!

File: apps/editorial_features/components/eoy/stylesheets/body.styl [hunk 2]
```diff
@@ -6,6 +6,10 @@ flex-display(justify = space-between, align = center)
 body
   width 100vw
   overflow-x hidden
+
+video::-webkit-media-controls
+  display none !important
+
```
Comment (kanaabe) [inline apps/editorial_features/components/eoy/stylesheets/body.styl]: Yep!

**Comment by kanaabe** [comment]
Soo I added back `IS_MOBILE` for accessibility. @craigspaeth made a good suggestion to use a more robust user agent checker like https://github.com/3rd-Eden/useragent. They didn't have a nice way to check if something is mobile, and I thought [this feature request](https://github.com/3rd-Eden/useragent/issues/87) was pretty similar to what we have so I decided to punt on that.

**Comment by kanaabe** [comment]
hey @joelauerbach  -- would you mind taking a look at the EOY feature (/eoy-2016) and making sure all the events are tracked properly from this PR? I want to make sure we're not missing any contexts or other metadata. Let me know if I should make any adjustments! Thanks.

**Comment by joelauerbach** [comment]
@kanaabe Thanks! A few notes:

1. The TOC impression doesn't fire from the bottom of the main page (it does fire from the bottom of individual articles).

2. The TOC impression also fires at the top of individual articles - is this from the header? Note that the event fires before the header is hovered over (i.e., before it's visible). It also seems to fire a bunch of extra times (5 in my tests).

image

3. There's a `recently viewed artworks` event that fires as soon as the scroll bar is moved (on main page and individual articles). Not sure what that is.

image

4. On https://staging.artsy.net/article/artsy-editorial-eoy-2016, the TOC impression seems to fire a bunch of extra times - it looks like maybe once for every item in the TOC (4), plus an additional 2 that I can't account for. The latter 2 have an article ID attached, but the first 4 don't.

image

And as you mentioned, we don't have the primary/secondary logos or the CTA container yet - let me know once those are up and I'll check them out. Thanks again :)

**Pull request merged**